analyze.sh

```diff
@@ -1,9 +1,23 @@
 #!/bin/sh
 RESULTS_DIR=${RESULTS_DIR:-"${@}"}
+
 echo "-----"
 echo "Benchmark finished. Detailed results are located in: ${RESULTS_DIR}"
+
 docker run --name analyzer --rm \
 	-v "${PWD}/analyze:/analyze:ro" \
 	-v "${PWD}/${RESULTS_DIR}:/reports:ro" \
 	ruby:2.7-buster ruby /analyze/results_analyze.rb reports ||
 	exit 1
+
+cat > ${RESULTS_DIR}/bench.params << EOF
```

Comment:
It's already done in `bench.sh`, you could add the warmup param [there](https://github.com/LesnyRumcajs/grpc_bench/blob/master/bench.sh#L56)

Reply:
@brunoborges sure, makes sense